static/app/utils/replays/hooks/useReplayOnboarding.tsx

```diff
@@ -21,34 +24,34 @@ function getProjectList(selectedProjects: PageFilters['projects'], projects: Pro
   return selectedProjects.map(id => projectsByProjectId[id]).filter(Boolean);
 }
 
-export function useShouldShowOnboarding() {
+export function useHaveSelectedProjectsSentAnyReplayEvents() {
```

Comment:
It's a bit wordy but describes it well and can't think of anything more succinct.

Reply:
yeah i don't like it. but the old name wasn't doing it for me :(